### PR TITLE
Implement VAMC data layer for links within accordions

### PR DESCRIPTION
**Pull request by brangi**
## Description
Issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/21556

Screen Shot 2021-05-03 at 7 09 08 PM

### CODE REVIEW

File: src/site/facilities/health_service.drupal.liquid [hunk 1]
```diff
@@ -25,8 +25,21 @@
...
-                            {% if facility != empty %}<li class="vads-u-margin-bottom--2">
-                                <a href="{{ facility.entityUrl.path }}">{{ facility.title }}</a></li>{% endif %}
+                            {% if facility != empty %}
+                                <li class="vads-u-margin-bottom--2">
+                                    <a
+                                        onclick="recordEvent(
+                                            {
+                                                'event': 'nav-accordion-link-click',
+                                                'accordion-parent-label': '{{ serviceTaxonomy.name }}',
+                                                'accordion-child-label': '{{ serviceTaxonomy.fieldAlsoKnownAs }}',
```
Comment (mpelzsherman) [inline src/site/facilities/health_service.drupal.liquid:35]: If `serviceTaxonomy.fieldAlsoKnownAs` is null, do we need to substitute the string "undefined"? Or does that happen automatically?

**Review by mpelzsherman — APPROVED** [review]
Please check the comment re: "undefined".

We should probably add some unit tests but that can be done later.

**Comment by brangi** [comment]
> Please check the comment re: "undefined".
> 
> We should probably add some unit tests but that can be done later.

alright, I can check that

**Review by timwright12 — APPROVED** [review]

File: src/site/facilities/health_service.drupal.liquid [hunk 2]
```diff
@@ -25,8 +25,21 @@
...
+                                                'accordion-section-label': '{{ sectionName }}'
+
+                                            })"
+                                        href="{{ facility.entityUrl.path }}">{{ facility.title }}
+                                    </a>
+                                </li>
+                            {% endif %}
```
Comment (mpelzsherman) [inline src/site/facilities/health_service.drupal.liquid:42]: Should we create a tech-debt ticket to convert these to `va-accordion`?

**Pull request merged**